ocaml/xapi/repository.ml

```diff
@@ -750,8 +750,12 @@ let apply_updates' ~__context ~host ~updates_info ~livepatches ~acc_rpm_updates
   (* Evaluate recommended/pending guidances *)
   let recommended_guidances, pending_guidances =
     let recommended_guidances' =
-      eval_guidances ~updates_info ~updates:acc_rpm_updates ~kind:Recommended
-        ~livepatches:successful_livepatches ~failed_livepatches
+      (* EvacuateHost will be applied before applying updates *)
+      List.filter
+        (fun g -> g <> Guidance.EvacuateHost)
+        (eval_guidances ~updates_info ~updates:acc_rpm_updates ~kind:Recommended
+           ~livepatches:successful_livepatches ~failed_livepatches
+        )
```

Comment:
Would this way look clearer?
```suggestion
      (* EvacuateHost will be applied before applying updates *)
      eval_guidances ~updates_info ~updates:acc_rpm_updates ~kind:Recommended
           ~livepatches:successful_livepatches ~failed_livepatches
      |> List.filter (fun g -> g <> Guidance.EvacuateHost)
```

Reply:
Changed this way also for the following lines of code to keep the same style.